x-pack/plugins/index_management/__jest__/client_integration/home/indices_tab.test.ts

```diff
@@ -246,13 +255,18 @@ describe('<IndexManagementHome />', () => {
       await actions.clickManageContextMenuButton();
       await actions.clickContextMenuOption('closeIndexMenuButton');
 
-      // A refresh call was added after closing an index so we need to check the second to last request.
```

Comment:
Should we keep part of this comment explaining that a refresh request will be made after closing an index?